### PR TITLE
App strings reworked

### PR DESCRIPTION
**Pull request by comradekingu**
This takes more time for me.
All the reverted strings that were edited directly are improvements.
Happy to explain why for each one if needed.

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -65,14 +65,14 @@
...
-    <string name="no_phone_number_found">No phone number has been found</string>
+    <string name="no_phone_number_found">Could not find any phone number</string>
 
     <!-- Events -->
     <string name="birthday">Birthday</string>
     <string name="anniversary">Anniversary</string>
 
     <!-- Favorites -->
-    <string name="no_favorites">Seems like you haven\'t added any favorite contacts yet.</string>
+    <string name="no_favorites">Add some favorite contacts to see them here.</string>
```
Comment (comradekingu) [inline app/src/main/res/values/strings.xml:75]: Not sure

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -84,7 +84,7 @@
     <string name="export_contacts">Export contacts</string>
     <string name="import_contacts_from_vcf">Import contacts from a .vcf file</string>
     <string name="export_contacts_to_vcf">Export contacts to a .vcf file</string>
-    <string name="target_contact_source">Target contact source</string>
+    <string name="target_contact_source">Target contact-source</string>
```
Comment (comradekingu) [inline app/src/main/res/values/strings.xml:87]: What does this mean?

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -107,62 +107,56 @@
     <string name="ringtone">Ringtone</string>
 
     <!-- Confirmation dialog -->
-    <string name="delete_from_all_sources">The contact will be removed from all contact sources.</string>
+    <string name="delete_from_all_sources">Remove contact from all contact sources?</string>
```
Comment (comradekingu) [inline app/src/main/res/values/strings.xml:110]: Not sure

**Comment by tibbi** [comment]
cmon, english strings are good, you are supposed to translate the other languages. Thanks

**Comment by comradekingu** [comment]
@tlbbl
And you seem to think my changes somehow aren't fine?

I can't seem to understand how you came to think I found as many errors as the strings that aren't reverted,
when this is more of the same.
Nevertheless I told you the other changes are also fixes, but you don't care to ask why.
This has plain errors, no consistency, and uses paradigms from the 90s.
The short_desc is too long for the new playstore restrictions, and the long_desc has not only errors and bad language, but totally disregards the attention span of the user. Look at any other description from a major app and it won't do any of that. 
Why have some descriptions rewritten, but then not this?
It has the same amount of info, in about half the amount of text, in a logical manner.

Should I comment everything, how hard could this be?

**Comment by tibbi** [comment]
Im not interested in adjusting my strings to Weblate, if they don't make sense..

**Pull request closed without merging**